invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Sections/SectionsWrapper.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import SectionServices from './SectionServices';
+import { Container } from 'semantic-ui-react';
+import SectionTags from './SectionTags';
+import { document as documentApi } from '../../../../common/api';
+import { FrontSiteRoutes } from '../../../../routes/urls';
+import { BookGroup } from '../BookGroup';
+
+export default class SectionsWrapper extends Component {
+  render() {
+    if (this.props.sections && !isEmpty(this.props.sections)) {
+      console.log(this.props, '===========');
+      return (
+        <Container fluid className={'fs-landing-page-section-wrapper'}>
+          {this.props.sections.map(Section => {
+            return <Section className={'fs-landing-page-section'} />;
+          })}
+        </Container>
+      );
+    } else {
```

Comment:
Split the logic and do the checks outside render

Reply:
done